contracts/libraries/pool/BurnCall.sol

```diff
@@ -24,21 +24,28 @@ library BurnCall {
         mapping(int24 => IRangePoolStructs.Tick) storage ticks,
         IRangePoolStructs.Sample[65535] storage samples
     ) external returns (IRangePoolStructs.BurnCache memory) {
+        if (params.burnPercent > 1e38) params.burnPercent = 1e38;
         (
             cache.position,
             cache.amount0,
-            cache.amount1
+            cache.amount1,
+            cache.tokenBurned
         ) = Positions.update(
```

Comment:
`cache.tokenBurned` is passed into `Positions.remove()` so we know how much liquidity they own for that `Position`.